src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -155,22 +160,19 @@ public StreamingResponseBody streamEvents(
                 outputStream.flush(); // Flush status code to client
 
                 eventStream.streamEvents(connectionReady);
-            }
-            catch (final NoSuchEventTypeException e) {
+            } catch (final NoSuchEventTypeException e) {
                 writeProblemResponse(response, outputStream, NOT_FOUND, "topic not found");
-            }
-            catch (final NakadiException e) {
+            } catch (final NakadiException e) {
                 LOG.error("Error while trying to stream events. Respond with SERVICE_UNAVAILABLE.", e);
                 writeProblemResponse(response, outputStream, e.asProblem());
-            }
-            catch (final InvalidCursorException e) {
+            } catch (final InvalidCursorException e) {
                 writeProblemResponse(response, outputStream, PRECONDITION_FAILED, e.getMessage());
-            }
-            catch (final Exception e) {
+            } catch (final IllegalScopeException e) {
```

Comment:
BTW, why you catch this exception? As I see you added the handler for it to ExceptionHandling class. So it should be automatically converted to response, right?